src/registrar/templates/includes/footer.html

```diff
@@ -1,4 +1,5 @@
 {% load static %}
+{% load field_helpers url_helpers %}
```

Comment:
**question (non-blocking)**: It's not a big deal, but I don't see where you need the field_helpers here? With what you added, I think that this will work with just
```suggestion
{% load url_helpers %}
```